jobs/nicheny-django-developer.html

```diff
@@ -0,0 +1,40 @@
+---
+title: Freelance Django Developer
+company: Niche
+location: New York, NY
```

Comment:
New York is a big place - you can specify a neighborhood or even a zip code if you want candidates to know where your office is.